utils/get_supported_scylla_base_versions.py

```diff
@@ -95,8 +95,18 @@ def get_supported_scylla_base_versions(self, supported_versions) -> list:  # pyl
                     # The dest version is a minor release of latest enterprise release
                     ent_base_version.append(version)
                 else:
-                    # Choose the last two releases as upgrade base
-                    ent_base_version += ent_release_list[idx-1:][:2]
+                    lts_version = re.compile(r'\d{4}\.1')  # lts = long term support
+                    sts_version = re.compile(r'\d{4}\.2')  # sts = short term support
+
+                    if sts_version.search(version) or ComparableScyllaVersion(version) < '2023.1':
+                        # we need to support last LTS version
+                        ent_base_version += ent_release_list[idx - 1:][:2]
+                    elif lts_version.search(version):
+                        # we need to support last version + last LTS version
+                        idx = 2 if idx == 1 else idx
+                        ent_base_version += ent_release_list[idx - 2:][:2]
+                    else:
+                        LOGGER.warning(f'enterprise version format not the default - {version}')
```

Comment:
The `precommit` check fails on this line demanding usage of the `%` lazy formatting:

```
23:03:33  ************* Module utils.get_supported_scylla_base_versions
23:03:33  utils/get_supported_scylla_base_versions.py:109:24: W1203: Use lazy % or .format() formatting in logging functions (logging-fstring-interpolation)
```

Reply:
fixed